### PR TITLE
Update Office 365 TI connection office-365-ti.md

### PR DESCRIPTION
**Pull request by ajaj-shaikh**
Deprecating MDE setting button.

### CODE REVIEW

File: microsoft-365/security/office-365-security/office-365-ti.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ This feature is only available if you have an active Office 365 E5 subscription
 When you turn on this feature, you'll be able to incorporate data from Microsoft Defender for Office 365 into Microsoft 365 Defender to conduct a comprehensive security investigation across Office 365 mailboxes and Windows devices.
 
 > [!NOTE]
-> You'll need to have the appropriate license to enable this feature.
+> This setting is no longer functional in security.microsoft.com portal and will be deprecated soon. Users can still see MDO data in security portal and are advised not to use this setting it.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/office-365-ti.md]:
```suggestion
> This setting is no longer functional in the Microsoft 365 Defender portal at <https://security.microsoft.com> and will be deprecated soon. Users can still see Microsoft Defender for Office 365 data in the security portal and are advised not to use this setting. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ajaj-shaikh, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ajaj-shaikh** [comment]
@MSFTTracyP can you please merge this PR?

**Comment by yogkumgit** [comment]
@MSFTTracyP Gentle reminder

**Pull request merged**